src/test/java/uk/gov/hmcts/probate/controller/DocumentControllerTest.java

```diff
@@ -345,4 +349,17 @@ public void generateGrantDraftReissueGrantOfRepresentation() throws Exception {
                 .andExpect(content().string(containsString("data")));
 
     }
+
+    @Test
+    public void displayAttachScanDocErrorIfUsedFromUI() throws Exception {
+
+        String solicitorPayload = testUtils.getStringFromFile("solicitorPayloadNotifications.json");
+
+        ResultActions resultActions = mockMvc.perform(post("/document/attach-scanned-docs-error")
```

Comment:
whats with the ResultActions? its not used anywhere and it looks like youre just getting the body of the message, could the methods that are already used in the other tests be used here too??

Reply:
I've removed ResultActions, as you said it wasn't used.